tafinder/app/models/term.rb

```diff
@@ -1,6 +1,6 @@
 class Term < ActiveRecord::Base
   has_many :rankings, dependent: :nullify
```

Comment:
@swm93 I agree with all points you made. Would it work better if we gave Application their own unique id's instead of using student_id and term to differentiate them? and then just make sure student_id + term_id are unique?